packages/web-components/src/components/button/button.ts

```diff
@@ -23,7 +23,7 @@ const { prefix, stablePrefix: c4dPrefix } = settings;
  * Button.
  *
  * @element c4d-button
- * @csspart button.
+ * @csspart button - The button. Usage `c4d-button::part(button)`.
```

Comment:
This seems like a good pattern to go forward with! One minor nit:
```suggestion
 * @csspart button - The button. Usage: `c4d-button::part(button)`.
```